README.md

```diff
@@ -152,9 +152,10 @@ Comming soon!!!
 
 👤 Luis Emilio Rojas
 
-- GitHub: [@githubhandle](https://github.com/lerfast)
+- GitHub: [@githublerfast](https://github.com/lerfast)
 - Twitter: [@twitterhandle](https://twitter.com/lerfast)
 - LinkedIn: [LinkedIn](https://www.linkedin.com/in/luis-emilio-rojas-rojas-6ba90a8a/)
+- GitHub online page:[Github-page](https://lerfast.github.io) 
 
```

Comment:
- [ ] The link that you attached as the deployed version of your application displays a _**Hello World**_ project. Kindly attach the correct link to the deployed project. 👍 😎

Reply:
Yes off course, thats because i didnt merge waiting to be aproved, should I merge?